src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for data that requires the ability to be printed
+ * Guarantees : Existence of method to print data as string
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Missing header comment. All non-trivial classes and methods should have javadoc format header comments.